papers/haskell-exchange-2019/stg-grin/stg-grin.abstract

```diff
@@ -0,0 +1,5 @@
+A comparison between STG and GRIN
```

Comment:
I think this shouldn't be part of the PR. :)

Reply:
Done.